pkgs/top-level/all-packages.nix

```diff
@@ -5672,6 +5672,8 @@ with pkgs;
     wlroots = wlroots_0_15;
   };
 
+  hyprshade = python311Packages.callPackage ../applications/window-managers/hyprwm/hyprshade { };
```

Comment:
It needs precisely this version of Python?

Reply:
I was getting errors otherwise. It's defined [here](https://github.com/loqusion/hyprshade/blob/main/pyproject.toml) under `[project]` there is `requires-python = ">=3.11,<4.0"`.

Reply:
Ok then.